pkg/controller/job-controller.go

```diff
@@ -230,19 +241,16 @@ func (c *JobController) SyncHandler(key string) (Result, error) {
 	}
 	err = c.k8sClient.Get(ctx, client.ObjectKeyFromObject(tenant), tenant)
 	if err != nil {
-		jobCR.Status.Phase = miniojob.MinioJobPhaseError
-		jobCR.Status.Message = fmt.Sprintf("Get tenant %s/%s error:%v", jobCR.Spec.TenantRef.Namespace, jobCR.Spec.TenantRef.Name, err)
-		err = c.updateJobStatus(ctx, &jobCR)
-		return WrapResult(Result{}, err)
+		return WrapResult(Result{}, fmt.Errorf("get tenant %s/%s error:%w", jobCR.Spec.TenantRef.Namespace, jobCR.Spec.TenantRef.Name, err))
```

Comment:
It's common to put a space before `%w` (see [Go blog](https://go.dev/blog/go1.13-errors)):
```suggestion
		return WrapResult(Result{}, fmt.Errorf("get tenant %s/%s error: %w", jobCR.Spec.TenantRef.Namespace, jobCR.Spec.TenantRef.Name, err))
```